test/screenshot/README.md

```diff
@@ -15,7 +15,8 @@ curl https://sdk.cloud.google.com | bash
 Create GCP service account key:
 
 - Go to the [Create Service Account Key Page](https://console.cloud.google.com/apis/credentials/serviceaccountkey).
-- Make sure you're logged into `material-components-web` project.
+- Make sure you're logged into the **Material Components Web Demo** project.
+  - If you can't find the project, check under the ["No organization" group](./gcp-no-organization-modal.png).
```

Comment:
I think screenshot is not required, people will figure it out :)

Reply:
I'd prefer to keep it to make it super obvious. It's a click-through link so it doesn't show up on the page.
<img width="1020" alt="Screen Shot 2019-11-21 at 10 54 32 AM" src="https://user-images.githubusercontent.com/5939574/69367622-76996180-0c4d-11ea-9c03-9bc7d1f3a0ed.png">